src/Graphics/Color.cs

```diff
@@ -14,942 +14,1121 @@
+
+		Vector3 scrollingRGBForEvenSlices = new Vector3(b, tempRGB.Z, tempRGB.X); // (V, Temp Blue, Temp Red) for even slices (0, 2, 4)
+		Vector3 scrollingRGBForOddSlices = new Vector3(tempRGB.Y, b, tempRGB.X);  // (Temp Green, V, Temp Red) for odd slices (1, 3, 5)
+		Vector3 scrollingRGB = Vector3.Lerp(scrollingRGBForEvenSlices, scrollingRGBForOddSlices, isOddSlice);
+
+		float IsNotFirstSlice = float.Clamp(threeSliceSelector, 0f, 1f);        // 1 if NOT the first slice (true for slices 1 and 2)
+		float IsNotSecondSlice = float.Clamp(threeSliceSelector - 1f, 0f, 1f);  // 1 if NOT the first or second slice (true only for slice 2)
+
+		Vector3 color = Vector3.Lerp(
+			scrollingRGB,
+			Vector3.Lerp(
+				new Vector3(scrollingRGB.Z, scrollingRGB.X, scrollingRGB.Y),
+				new Vector3(scrollingRGB.Y, scrollingRGB.Z, scrollingRGB.X),
+				IsNotSecondSlice
+			),
+			IsNotFirstSlice
+		);
+
+		return new Color(color.X, color.Y, color.Z, 1f);
+	}
+
+	public static Color FromHSV(int r, int g, int b)
+	{
+		return FromHSV(r / 255f, g / 255f, b / 255f);
+	}
+
+	/// <summary>
+	/// Multiply <see cref="Color"/> by value.
+	/// </summary>
+	/// <param name="value">Source <see cref="Color"/>.</param>
+	/// <param name="scale">Multiplicator.</param>
+	/// <returns>Multiplication result.</returns>
+	public static Color operator *(Color value, float scale)
+	{
+		return new Color(
+			value.R * scale / 255f,
+			value.G * scale / 255f,
+			value.B * scale / 255f,
+			value.A * scale / 255f
+		);
+	}
+
+	/// <summary>
+	/// Returns a <see cref="String"/> representation of this <see cref="Color"/> in the format:
+	/// {R:[red] G:[green] B:[blue] A:[alpha]}
+	/// </summary>
+	/// <returns><see cref="String"/> representation of this <see cref="Color"/>.</returns>
+	public override string ToString()
+	{
+		StringBuilder sb = new StringBuilder(25);
+		sb.Append("{R:");
+		sb.Append(R);
+		sb.Append(" G:");
+		sb.Append(G);
+		sb.Append(" B:");
+		sb.Append(B);
+		sb.Append(" A:");
+		sb.Append(A);
+		sb.Append("}");
+		return sb.ToString();
+	}
+
+	/// <summary>
+	/// Returns a hexadecimal <see cref="String"/> representation of this <see cref="Color"/> in the format:
+	/// 0x[R][G][B][A]
+	/// <para>Thus, it's in Big Endian format.</para>
+	/// </summary>
+	/// <returns><see cref="String"/> representation of this <see cref="Color"/>.</returns>
+	public string ToHexString()
+	{
+		StringBuilder sb = new StringBuilder(10);
+		sb.Append("#");
+		sb.Append(R.ToString("X2"));
+		sb.Append(G.ToString("X2"));
+		sb.Append(B.ToString("X2"));
+		sb.Append(A.ToString("X2"));
+		return sb.ToString();
+	}
+
+	public static implicit operator VertexStructs.Ubyte4Norm(Color color) => new VertexStructs.Ubyte4Norm
+	{
+		X = color.R,
+		Y = color.G,
+		Z = color.B,
+		W = color.A
+	};
+
+	public static implicit operator FColor(Color color) => new FColor
+	{
+		R = color.R / 255f,
+		G = color.G / 255f,
+		B = color.B / 255f,
+		A = color.A / 255f
+	};
+
+
+#region Built-In Colors
```

Comment:
Let's not do regions please.

Reply:
Interesting, I wasn't aware of their reputation as a code smell. It seemed useful for the "minimap" in VS Code to visually see where you have to scroll to, but I've removed it now.